### DISCUSSION TITLE
Inherit target values

### DISCUSSION THREAD

**Discussion by romanzenka** [ropensci/targets#725] [Help] [answered]
I am making a series of pipelines that depend on each other.

I found inheritance in `_targets.yaml` for settings of the pipeline, but I was dismayed to see it did not inherit data.

Currently the solutions described in the manual are:
- manually load these objects yourself, duplicating them in the pipeline cache folder (+ adding 2 targets per each dependency)
- reference the internals of the _targets/objects storage directly, breaking encapsulation

Can there be a better way of inheriting data from another pipeline:
- without duplicating the data on disk
- without explicitly spelling out every dependence
- not breaking encapsulation and digging into internals of `targets` 
?

**Comment by wlandau** [comment, marked as answer]
@nsheff maintains a package called [`unitar`](https://github.com/databio/unitar) that I believe does this. `targets` itself does not have a built-in way to inherit data among multiple projects.

**Reply by romanzenka** [reply]
Thank you, that will work!

I think this functionality would be best baked into `targets` though... I went through the source code of that package and there is not that much in there, at the cost of starting its own naming convention for functions, and going straight into `targets` internals. I can use it for now, but I would suggest updaring `targets` so this relatively basic functionality that is already half-implemened can be used out of the box, with all the quality and compatibility assurances.
